projects/packages/search/src/instant-search/components/dom-event-handler.jsx

```diff
@@ -190,6 +191,14 @@ export default class DomEventHandler extends Component {
 		}
 	};
 
+	fixBodyOpacity = () => {
+		if ( '0' === document.body.style.opacity ) {
```

Comment:
I'll note that if the body opacity is set to `0` by CSS alone (as per the PR description), `document.body.style.opacity` will be empty, and this won't trigger, so I suspect there's some JS or inline CSS in the theme that's setting the opacity as well.

It might be worth always setting the opacity to `1`, or at least doing so if the string is empty.

Reply:
Thanks, I've changed it back to the initial implementation.